.github/workflows/dev-image-update.yaml

```diff
@@ -88,12 +82,6 @@ jobs:
           add-tag \
           gcr.io/${{ env.gcr_google_project }}/jade-data-repo-ui:"${GCR_TAG}" \
           gcr.io/${{ env.gcr_google_project }}/jade-data-repo-ui:"${{ steps.uiprevioustag.outputs.tag }}" --quiet
-      - name: 'Check and edit Helm definition for dev'
-        uses: broadinstitute/datarepo-actions/actions/main@0.67.0
-        with:
-          actions_subcommand: 'deploytagupdate'
-          helm_env_prefix: dev
-          helm_imagetag_update: ui
```

Comment:
This actions appears to also tag the images with the new version - https://github.com/broadinstitute/datarepo-actions/blob/master/actions/main/src/deploytagupdate.sh#L11
We can just pull this into a new step (We're moving away from datarepo-actions when possible, anyway)

Reply:
I did not see that, that's a really good catch -- I thought tag just meant git tag

Reply:
Hmm... Maybe we don't need it. Looks like it already has the version tagged image (e.g. 1.538.0), but we're adding a githash-develop tag to the image as well.. I wonder if we still need that.

Reply:
Well, we are still using it as the tag. https://github.com/broadinstitute/terra-helmfile/blob/master/values/app/datarepo/live/dev.yaml#L79

One day, we should swap to using the version, but let's just change one thing at a time.

Reply:
It seems like the only effect that we actually want from that section is https://github.com/broadinstitute/datarepo-actions/blob/master/actions/main/src/deploytagupdate.sh#L17 -- it takes `gcr.io/broad-jade-dev/jade-data-repo-ui:${GCR_TAG}` and adds `gcr.io/broad-jade-dev/jade-data-repo-ui:${GCR_TAG}-develop`. I can add this to the command above

Reply:
That's perfect! Thanks!